modules/administration/pages/public-cloud.adoc

```diff
@@ -0,0 +1,351 @@
++
+----
+$ echo "${local_address} suma.cloud.net suma" >> /etc/hosts
+----
++
+You can find the local address by checking your public cloud web console, or from the command line :
++
+* Amazon EC2 instance:
++
+----
+$ ec2metadata --local-ipv4
+----
+* Google Compute Engine:
++
+----
+$ gcemetadata --query instance --network-interfaces --ip
+----
++
+* Microsoft Azure:
++
+----
+$ azuremetadata --internal-ip
+----
+
+=== Set up DNS resolution
+
+You will need to update the DNS records for the instance within the DNS service of your network environment.
+Refer to the cloud service provider documentation for detailed instructions:
+*** http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/vpc-dns.html[ DNS setup on Amazon EC2]
+*** https://cloud.google.com/compute/docs/networking[ DNS setup on Google Compute Engine]
+*** https://azure.microsoft.com/en-us/documentation/articles/dns-operations-recordsets[ DNS setup on Microsoft Azure]
+
+If you run a {productname} Server instance, you can run YaST after the instance is launched to ensure the external storage is attached and prepared correctly, and that DNS resolution is set up as described:
+
+----
+$ /sbin/yast2 susemanager_setup
+----
+
+
+////
+
+No need to duplicate this, since it exists within the docs suite. LKB 2019-05-29
+
+Note that the setup of {productname} from this point forward does not differ from the documentation in the https://www.suse.com/documentation/suse_manager[SUSE Manager Guide].
+
+The {productname} setup procedure in YaST is designed as a one pass process with no rollback or cleanup capability.
+Therefore, if the setup procedure is interrupted or ends with an error, it is not recommended that you repeat the setup process or attempts to manually fix the configuration.
+These methods are likely to result in a faulty {productname} installation.
+If you experience errors during setup, start a new instance, and begin the setup procedure again on a clean system.
+
+If you receive a message that there is not enough space available for setup, ensure that your root volume is at least 20GB and double check that the instructions in <<using-separate-storage-volume>> have been completed correctly.
+
+{productname} Server for the public cloud comes with a bootstrap data module pre-installed.
+The bootstrap module contains optimized package lists for bootstrapping instances started from {sle} images published by {suse}.
+If you intend to register such an instance, when you creatr the bootstrap repository run the [command]``mgr-create-bootstrap-repo`` script using this command, to create a bootstrap repository suitable for {sle} 12 SP1 instances.
+
+----
+$ mgr-create-bootstrap-repo --datamodule=mgr_pubcloud_bootstrap_data -c SLE-12-SP1-x86_64
+----
+
+
+See https://www.suse.com/documentation/suse-manager-3/book.suma.getting-started/data/create_tools_repository.html[Creating the SUSE Manager Tools Repository] for more information on bootstrapping.
+
```

Comment:
This section about the bootstrap module is obsolete and should be removed.

Reply:
It's been commented out.